client/src/components/applicants/ApplicantModalDropzone.js

```diff
@@ -0,0 +1,152 @@
+import React from 'react'
+import { connect } from 'react-redux'
+import PropTypes from 'prop-types'
+import { addComment } from '../../redux/actions/actions'
+import { Button, Chip, TextField } from '@material-ui/core'
+import { AttachFile } from '@material-ui/icons'
+import Dropzone from 'react-dropzone'
+
+class ApplicantModalDropzone extends React.Component {
+
+  constructor(props) {
+    super(props)
+    this.state = {
+      inputError: null,
+      comment: '',
+      attachments: []
+    }
+  }
+
+  handleSubmit = () => {
+    if (!this.state.comment.trim()) {
+      this.setState({ inputError: 'Cannot send empty comment' })
+    } else {
+      this.props.addComment(this.state.comment, this.props.applicant.id, this.state.attachments)
+      this.setState({ comment: '', inputError: null, attachments: [] })
+    }
```

Comment:
ApplicantModalDropzone doesn't have a props.applicant. Plus the unneeded inputError: null